router/build.gradle

```diff
@@ -26,32 +30,31 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
-    kotlinOptions {
-        jvmTarget = '1.8'
-    }
     buildFeatures {
         compose true
     }
     composeOptions {
-        kotlinCompilerVersion "1.3.70-dev-withExperimentalGoogleExtensions-20200424"
-        kotlinCompilerExtensionVersion "0.1.0-dev14"
+        kotlinCompilerVersion "1.4.0-dev-withExperimentalGoogleExtensions-20200720"
+        kotlinCompilerExtensionVersion "$compose_version"
     }
-    packagingOptions {
-        excludes = []
+    kotlinOptions {
+        jvmTarget = "1.8"
+        freeCompilerArgs += ["-Xallow-jvm-ir-dependencies", "-Xskip-prerelease-check"]
     }
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$project.ext.kotlin_version"
```

Comment:
Sorry, I was trying to get it to compile, and thought it was a dependency error, when it was really my fault for changing the core module plugin to application instead of library which resulted in an error which wasn't super clear.
Do you want me to just revert this whole change? If the code compiles and runs without the first 2 lines why are they still needed?

Reply:
You still need stdlib for sure, but first one can be removed, yes :)